src/Files.App (Package)/Package.appxmanifest

```diff
@@ -16,7 +16,7 @@
     <Identity
         Name="FilesDev"
         Publisher="CN=Files"
-        Version="3.6.2.0" />
+        Version="3.6.3.0" />
```

Comment:
Should this change be merged into main?

Reply:
I'm currently testing this commit in the preview branch.